test/08_test_layout.jl

```diff
@@ -283,55 +270,82 @@ end
 end
 
 @testset "Layout - GRID" begin
-    g = grid(; layout = (3, 3), pad = (0, 0))
-    @test g.measure.w == 48
-    @test g.measure.h == 27
+    g = grid(layout = (3, 3), pad = (0, 0))
+    @test size(g.measure) == (48, 27)
 
-    g = grid(; layout = (3, 3))
-    @test g.measure.w == 48
-    @test g.measure.h == 27
+    g = grid(layout = (3, 3))
+    @test size(g.measure) == (48, 27)
 
-    g = grid(; layout = (3, 3), pad = 2)
-    @test g.measure.w == 52
-    @test g.measure.h == 31
+    g = grid(layout = (3, 3), pad = 2)
+    @test size(g.measure) == (52, 31)
 
-    g = grid(; layout = (3, 3), pad = (5, 1))
-    @test g.measure.w == 58
-    @test g.measure.h == 29
+    g = grid(layout = (3, 3), pad = (5, 1))
+    @test size(g.measure) == (58, 29)
 
-    g = grid(; layout = (3, 3), pad = (5, 3))
-    @test g.measure.w == 58
-    @test g.measure.h == 33
+    g = grid(layout = (3, 3), pad = (5, 3))
+    @test size(g.measure) == (58, 33)
 
     # test passing renderables
     rens = repeat([PlaceHolder(10, 5)], 9)
 
     g = grid(rens; aspect = 1)
-    @test g.measure.w == 30
-    @test g.measure.h == 15
+    @test size(g.measure) == (30, 15)
 
-    g = grid(rens;)
-    @test g.measure.w == 50
-    @test g.measure.h == 10
+    g = grid(rens)
+    @test size(g.measure) == (50, 10)
 
     g = grid(rens; pad = (2, 1))
-    @test g.measure.w == 58
-    @test g.measure.h == 11
+    @test size(g.measure) == (58, 11)
 
     g = grid(rens; pad = (2, 1), aspect = 0.5)
-    @test g.measure.w == 22
-    @test g.measure.h == 29
+    @test size(g.measure) == (22, 29)
 
     g = grid(rens; pad = (2, 1), aspect = 1.5)
-    @test g.measure.w == 46
-    @test g.measure.h == 17
+    @test size(g.measure) == (46, 17)
 
     g = grid(
         rens;
         pad = (2, 1),
         aspect = (12, 12),
         placeholder = PlaceHolder(10, 5; style = "red"),
     )
-    @test g.measure.w == 34
-    @test g.measure.h == 17
+    @test size(g.measure) == (34, 17)
+
+    w, h = 20, 10
+    panels = collect(
+        Panel("{on_$c} {/on_$c}", style = "bold yellow", width = w, height = h) for
```

Comment:
I've added a `background` option to `Panel` if you want to have the panel border and content colored. `Panel(; background="on_red")` is the usage, it's not documented yet but it works.

Reply:
Mmmh, I can't get it to work.